datateam-parent-pom/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.mainstreethub</groupId>
+    <artifactId>dropwizard-scala-parent-pom</artifactId>
+    <version>1.0.2-1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>datateam-parent-pom</artifactId>
```

Comment:
Might want to rename as datateam-dropwizard-scala-parent-pom  (heinously long, I know, but it'd be more accurate)